fish_speech/train.py

```diff
@@ -1,8 +1,6 @@
 import os
-
-os.environ["USE_LIBUV"] = "0"
```

Comment:
libuv should be in front of everything

Reply:
Yeah so that environment variable is set before any libraries that might depend on it are imported.

Reply:
It should be set before pytorch, so at least before lightning / DDP imported

Reply:
Gotcha!